store/src/metrics.rs

```diff
@@ -61,13 +53,7 @@ impl Metrics {
 
         register!(
             key_insert_count,
```

Comment:
do we want to change the name in prometheus too?

Reply:
yep, renamed